src/probe/probe.c

```diff
@@ -121,6 +121,7 @@ static int probe_dma_buffer_init(struct probe_dma_buf *buffer, uint32_t size,
 	return 0;
 }
 
+#if !CONFIG_ZEPHYR_NATIVE_DRIVERS
```

Comment:
Is the [DEBUG] title still needed here ?

Reply:
Removed [DEBUG] from title.